crates/bevy_app/src/app_builder.rs

```diff
@@ -44,6 +44,22 @@ impl AppBuilder {
         &mut self.app.resources
     }
 
+    /// Starts the application (through main runner)
```

Comment:
```suggestion
    /// Start the application (through main runner)
```
Here and below. I think it reads a bit nicer.

Reply:
Can you also change the rest of these short descriptions from passive to active form?